src/app/app.module.ts

```diff
@@ -2,12 +2,15 @@ import { NgModule }       from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { FormsModule } from '@angular/forms';
 import { AppComponent }   from './app.component';
+import {RouterModule} from '@angular/router';
```

Comment:
spaces between { and } like the others and also can @angular imports but above our app imports also in alphabetical order :)